platform/linux/README.md

```diff
@@ -26,18 +26,20 @@ Optional: `libsqlite3-dev` (also available as vendored dependency).
 
 ### Building with Docker
 
-You can use a Docker container to build MapLibre Native. A `Dockerfile` that installes the required dependencies when the image is built is provided in this directory.
+You can use a Docker container to build MapLibre Native. Note that you cannot build using both Docker and host methods at the same time. If you want to switch, you need to clean the repository first, e.g. by using `git clean -dxfi -e .idea -e .clwb -e .vscode`
```

Comment:
This should be (made) possible.

Reply:
@louwers I don't think it can be -- the `cmake -B build ...` step creates absolute paths in the `build/` dir, so you should use just one method. If you can build locally, why would you use docker?

One workaround could be to set some env var inside docker container to force the output to go to another dir instead of `build/` or we could even remap that dir to some other location (although that's kinda tricky because mapping a dir and a subdir of the same dir is not a good practice i think)

Reply:
Can you build with Docker and locally if you use a different build directory? If yes, then that comment can be removed I think.